strings/salesman_replacement.json

```diff
@@ -0,0 +1,49 @@
+{
+
+    "salesman": {
+        "shuttle": "cungadero",
+        "clam": "pipis",
+        "clams": "pipis",
+        "egg": "pipis",
+        "eggs": "pipis",
+        "love": "hyperlink blocked",
+        "captain": "big shot",
+        "dollar": "kromer",
+        "dollars": "kromer",
+        "credit": "kromer",
+        "credits": "kromer",
+        "basement": "deep abyss",
+        "mech": "workout-ready body",
+        "exosuit": "workout-ready body",
+        "phazon": "NEO",
+        "bolts": "high quality encryption",
+        "jester": "clown around town",
+        "salesman": "number 1 rated saleman1997",
+        "soul": "HeartShapedObject",
+        "souls": "hearts",
```

Comment:
this is really uncommon terminology

Reply:
It's the most necessary for the reference to work, as its what hints that Spamton is more than just a meme for the first time.